File: official/docs/csharp/current/pickups/cancel.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            pickup = await client.Pickup.Cancel("pickup_...");

            Console.WriteLine(JsonConvert.SerializeObject(pickup, Formatting.Indented));
        }
    }
}

File: official/docs/golang/current/pickups/cancel.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func cancel() {
	client := easypost.New("EASYPOST_API_KEY")

	pickup, _ := client.CancelPickup("pickup_...")

	fmt.Println(pickup)
}
